spec/models/deidentified_client_spec.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DeidentifiedClient, type: :model do
+  describe '.visible_to' do
+    let!(:user_agency) { create(:agency, name: 'User Agency') }
+    let!(:other_agency) { create(:agency, name: 'Other Agency') }
+
+    let!(:user_with_edit_all_role) do
+      user = create(:user, agency: user_agency)
+      role = create(:role, name: 'editor_of_all', can_edit_all_clients: true)
+      user.roles << role
+      user
+    end
+
+    let!(:user_with_manage_deidentified_role) do
+      user = create(:user, agency: user_agency)
+      role = create(:role, name: 'deidentified_manager', can_edit_all_clients: false, can_manage_all_deidentified_clients: true)
+      user.roles << role
+      user
+    end
+
+    let!(:user_with_covid_pathways_role) do
+      user = create(:user, agency: user_agency)
+      role = create(:role, name: 'covid_pathway_viewer',
+                           can_edit_all_clients: false,
+                           can_manage_all_deidentified_clients: false,
+                           can_view_all_covid_pathways: true)
+      user.roles << role
+      user
+    end
```

Comment:
It looks like we're testing covid pathways (that no one uses any more) but not the standard `can_edit_diedentified_clients` which is what we just had issues with. Can we add that set of permissions to some tests?